kalite/testing/base_environment.py

```diff
@@ -245,3 +254,42 @@ def do_fake_registration():
     device = Device.get_own_device()
     device_zone = DeviceZone(device=device, zone=zone)
     device_zone.save()
+
+
+def _make_video(context):
+    lang_code = "en"
+    youtube_id = "my_cool_id"
+    item_dict = {
+        "title": "Subtitled Video",
+        "description": "A video with subtitles",
+        "available": True,
+        "kind": "Video",
+        "id": "video_with_subtitles",
+        "slug": "video_with_subtitles",
+        "path": "khan/video_with_subtitles",
```

Comment:
Yes, you would need to add a 'parent' field here that referenced the root node. It is the only node with the parent key as null.

There is currently no 'get' method in content_models.py but you could make one following the patterns here: https://github.com/learningequality/ka-lite/blob/0.16.x/kalite/topic_tools/content_models.py#L508 - obviously, no need for parse_item_data as we want the `Item` object itself.